drac_config

```diff
@@ -174,6 +176,9 @@ while [ $# -ne 0 ]; do
         "enableipmi")
             sh $libdir/ipmi.sh
             ;;
+        "disablehotspare")
+            sh $libdir/psu_redundancy.sh
+            ;;
```

Comment:
I think it would be better if this function was part of the "base" setup.

Reply:
It's added to the base profile and as a separate command :)